src/main/java/com/netuitive/ananke/statsd/client/request/EventRequest.java

```diff
@@ -0,0 +1,43 @@
+package com.netuitive.ananke.statsd.client.request;
+
+import com.netuitive.ananke.statsd.entity.Tag;
+import java.util.Date;
+import java.util.List;
+import lombok.Data;
+import lombok.experimental.Wither;
+
+/**
+ *
+ * @author john.king
+ */
+@Data
+@Wither
+public class EventRequest {
+
+    String title;
+    String text;
+    String alertType;
+    String aggregationKey;
+    String sourceTypeName;
+    Date dateHappened; 
+    String priority;
+    List<Tag> tags;
+    String hostname;
+
+    public EventRequest() {
+    }
+    
+
+    public EventRequest(String title, String text, String alertType, String aggregationKey, String sourceTypeName, Date dateHappened, String priority, List<Tag> tags, String hostname) {
```

Comment:
we should replace constructor with this many arguments with a builder pattern, i.e. new EventRequest().withTitle(...).withText(...).build();